kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -528,20 +555,19 @@ class GitOpsDeploymentManager extends React.Component {
 
   render() {
     const { appsList } = this.state;
-    const { getGitOpsRepoQuery } = this.props;
-    if (!appsList.length || getGitOpsRepoQuery.loading) {
+    if (!appsList.length) {
       return (
         <div className="flex-column flex1 alignItems--center justifyContent--center">
           <Loader size="60" />
         </div>
       );
     }
 
-    const gitopsRepo = getGitOpsRepoQuery.getGitOpsRepo;
+    const gitopsRepo = this.state.gitops;
     const activeStep = find(STEPS, { step: this.state.step });
     return (
       <div className="GitOpsDeploymentManager--wrapper flex-column flex1">
-        {gitopsRepo.enabled && this.state.step !== "action" ?
+        {gitopsRepo && gitopsRepo.enabled && this.state.step !== "action" ?
```

Comment:
just a suggestion, you can do this:
```suggestion
        {gitopsRepo?.enabled && this.state.step !== "action" ?
```